config/system/alt.yaml

```diff
@@ -22,6 +22,10 @@ interfaces:
     opts: -n 
     port: 3
 
+default_timeout_sec: 120
+
 # use vlan trigger
 run_trigger_type: VLAN
-default_timeout_sec: 120
+test_vlan_range:
```

Comment:
We've been avoiding the prefix "test" for this on principle since it's very confusing since it conflicts with the 200 other uses of the word "test" we have floating around -- so we should find something else to demark this.  Also "vlan" is redundant and unnecessarily specific, since we could, in theory, have other things that would trigger this.

I'm realizing now that the "run_trigger_type" param is also redundant, since it can be inferred from the existence of a vlan range.

How about we go with a slightly generalized form that looks like:

run_trigger:
  vlan_start: 2000
  vlan_end: 2100

and remove the "run_trigger_type" param and have that be inferred?

I'm somewhat future-proofing and generalizing to the world where there might be other and/or multiple triggers. Not sure what that might be, but conceptually there's nothing that would prevent different triggers from existing (e.g. based on MAC address, or some other parameter)... and keeping this more general would allow for a convenient bucket for all that.